### ISSUE TITLE
Deprecation of Loofah::HTML5::WhiteList

### ISSUE THREAD

**Issue by smap9** [a-ono/redmine_ckeditor#307]
After updating my redmine I got this message `redmine/plugins/redmine_ckeditor/init.rb:22: warning: constant Loofah::HTML5::WhiteList is deprecated`

It seems to be after updating flavorjones/loofah to https://github.com/flavorjones/loofah/releases/tag/v2.3.0

**Comment by joglomedia** [comment]
@smap9  did you have any solutions for this issue?

**Comment by tantic** [comment]
I think `WhiteList` has been replaced with `SafeList`, so changing
this in init.rb (line 22)
```
Loofah::HTML5::WhiteList::ALLOWED_PROTOCOLS.replace RedmineCkeditor.allowed_protocols
```
with
```
Loofah::HTML5::SafeList::ALLOWED_PROTOCOLS.replace RedmineCkeditor.allowed_protocols
```
worked for me

**Comment by tantic** [comment]
And maybe update rails_html_sanitizer to 1.3.0

**Comment by a-ono** [comment]
Fixed in version 1.2.3.
